vendors/espressif/boards/esp32/CMakeLists.txt

```diff
@@ -84,6 +90,40 @@ endif()
 afr_mcu_port(kernel)
 afr_glob_src(driver_src DIRECTORY "${esp_idf_dir}" RECURSE)
 
+if(ECC608_IN_USE)
+    set(mchp_dir "${AFR_VENDORS_DIR}/microchip")
+    set(ecc608a_dir "${mchp_dir}/secure_elements")
+
+    afr_glob_src(ecc608a_atacert_src RECURSE DIRECTORY ${ecc608a_dir}/lib/atcacert)
+
+    afr_glob_src(ecc608a_basic_src RECURSE DIRECTORY ${ecc608a_dir}/lib/basic)
+    afr_glob_src(ecc608a_crypto_src RECURSE DIRECTORY ${ecc608a_dir}/lib/crypto)
+    afr_glob_src(ecc608a_host_src RECURSE DIRECTORY ${ecc608a_dir}/lib/host)
+    afr_glob_src(ecc608a_pkcs11_src RECURSE DIRECTORY ${ecc608a_dir}/lib/pkcs11)
+
+    set(
+        ecc608a_hal_src
+            "${ecc608a_dir}/lib/hal/atca_hal.c"
+            "${ecc608a_dir}/lib/hal/hal_freertos.c"
+            "${ecc608a_dir}/lib/hal/hal_esp32_i2c.c"
+            "${ecc608a_dir}/lib/hal/hal_esp32_timer.c"
+            "${ecc608a_dir}/lib/hal/atca_hal.h"
+    )
+
+    afr_glob_src(ecc608a_src DIRECTORY ${ecc608a_dir}/lib)
+    list(REMOVE_ITEM ecc608a_src "${ecc608a_dir}/lib/atca_cfgs.c")
+
+    set(ecc608_include "${ecc608a_dir}/lib")
```

Comment:
Never used ?

Reply:
This config is set in a separate file, iot_pkcs11_secure_element.c